plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/DevBootableJarMojo.java

```diff
@@ -64,7 +64,8 @@ protected void configureCli(List<String> commands) {
     }
 
     private void configureScanner(Path deployments, List<String> commands) {
+        String deploymentPath = deployments.toString().replaceAll("\\\\", "\\\\\\\\");
```

Comment:
Can't you use replace instead of replaceAll?
```
deployments.toString().replace("\\", "\\\\");
```

Reply:
It is now with replace

Reply:
Thank-you. @jamezp , seems that you didn't run into this issue locally although the CLI should need this escaping. You confirm that it works for you?

Reply:
@jfdenise That is correct. It worked for me on the jaxrs example with the instructions on https://github.com/wildfly-extras/wildfly-jar-maven-plugin/blob/master/examples/dev-mode/README.md


```
[standalone@localhost:9990 /] /subsystem=deployment-scanner/scanner=wildfly-jar-for-dev-mode:read-resource
{
    "outcome" => "success",
    "result" => {
        "auto-deploy-exploded" => false,
        "auto-deploy-xml" => true,
        "auto-deploy-zipped" => true,
        "deployment-timeout" => 600,
        "path" => "C:\\Users\\jperkins\\projects\\wildfly-jar-maven-plugin\\examples\\jaxrs\\target\\deployments",
        "relative-to" => undefined,
        "runtime-failure-causes-rollback" => false,
        "scan-enabled" => true,
        "scan-interval" => 1000
    }
}
```


```
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: C:\Users\jperkins\apps\apache-maven-3.6.3\bin\..
Java version: 1.8.0_242, vendor: Oracle Corporation, runtime: C:\Users\jperkins\apps\java-1.8.0\jre
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```

Reply:
@doychin could you do the same than James did and see the value of the scanner path.

Reply:
I apologize @jfdenise. This fix is correct. I was testing on the wrong branch.

Reply:
```
[standalone@localhost:9990 /] /subsystem=deployment-scanner/scanner=wildfly-jar-for-dev-mode:read-resource
{
    "outcome" => "success",
    "result" => {
        "auto-deploy-exploded" => false,
        "auto-deploy-xml" => true,
        "auto-deploy-zipped" => true,
        "deployment-timeout" => 600,
        "path" => "D:wildfly-jar-maven-pluginexamplesjaxrstargetdeployments",
        "relative-to" => undefined,
        "runtime-failure-causes-rollback" => false,
        "scan-enabled" => true,
        "scan-interval" => 1000
    }

In my case path is invalid

Reply:
Could this be related to locale settings? I'm with Bulgarian settings.

In the code path is properly inserted into command string but later when run by the server it is not properly interpreted
```
[standalone@localhost:9990 scanner=wildfly-jar-for-dev-mode] :write-attribute(name=path,value=d:\temp\test)
{
    "outcome" => "success",
    "response-headers" => {
        "operation-requires-reload" => true,
        "process-state" => "reload-required"
    }
}

[standalone@localhost:9990 scanner=wildfly-jar-for-dev-mode] :resolve-path
{
    "outcome" => "success",
    "result" => "d:\\wildfly-jar-maven-plugin\\examples\\jaxrs\\        emp     est",
    "response-headers" => {"process-state" => "reload-required"}
}

[standalone@localhost:9990 scanner=wildfly-jar-for-dev-mode] :write-attribute(name=path,value=d:\\temp\\test)
{
    "outcome" => "success",
    "response-headers" => {
        "operation-requires-reload" => true,
        "process-state" => "reload-required"
    }
}

[standalone@localhost:9990 scanner=wildfly-jar-for-dev-mode] :resolve-path
{
    "outcome" => "success",
    "result" => "d:\\temp\\test",
    "response-headers" => {"process-state" => "reload-required"}
}

[standalone@localhost:9990 scanner=wildfly-jar-for-dev-mode]
```

As you can see when command is provided without double backslash it is treated as escape character